environment.yml

```diff
@@ -22,6 +22,8 @@ dependencies:
   - flake8
   - boto3-stubs-essential
   - pip:
+    - git+https://git@github.com/MetOffice/edr_server@CAP-340_add_collection_deserialisation
```

Comment:
@crbunney Now getting 'ModuleNotFoundError: No module named 'edr_server'', is the URL still wrong?

Reply:
oh yea, try this format: `https://github.com/{user}/{repository}/archive/{branch}.zip` 

e.g. https://github.com/MetOffice/edr_server/archive/CAP-340_add_collection_deserialisation.zip